CHANGES.rst

```diff
@@ -174,6 +174,10 @@ New Features
 
   - Enhanced text representation of ``WCS`` objects. [#3604]
 
+- The `astropy.tests.helper` module is now part of the public API (and has a
```

Comment:
Minor, annoying nitpick but this should use double-backticks.  The (current) changelog format does not assume any attempt to resolve API doc references (I'm hoping to address that in the new format eventually...)